Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -928,6 +968,47 @@ namespace AzFramework
         }
     }
 
+    auto SpawnableEntitiesManager::ProcessRequest(RetrieveTicketCommand& request) -> CommandResult
+    {
+        auto entitySpawnTicketIterator = m_entitySpawnTicketMap.find(request.m_ticketId);
+        if (entitySpawnTicketIterator == m_entitySpawnTicketMap.end())
+        {
+            if (m_ticketsPendingRegistration > 0)
+            {
+                // There are still tickets pending registration, which may hold the reference, so 
```

Comment:
nit: ...so?

Reply:
https://www.youtube.com/watch?v=wlwnbcxBuzI
Not sure what happened here, but will fix the comments.